handlers/accesscontrol/access-control.go

```diff
@@ -129,13 +127,11 @@ func (ac *AccessControlHandlersCollection) GetPlaybackAccessControlInfo(playback
 	ac.mutex.RUnlock()
 
 	if isExpired(entry) {
-		glog.Infof("Cache expired for playbackId=%v cacheKey=%v", playbackID, cacheKey)
```

Comment:
maybe we could log them on the verbose level or sth:
`glog.V(7).Infof("Cache expired for playbackId=%v cacheKey=%v", playbackID, cacheKey)`
in case we would need to debug caching